numpy/_core/tests/test_stringdtype.py

```diff
@@ -1212,14 +1235,16 @@ def test_unary(string_array, unicode_array, function_name):
     ("find", (None, "A")),
     ("index", (None, "e")),
     ("join", ("-", None)),
-    pytest.param("ljust", (None, 12), marks=unicode_bug_fail),
-    ("partition", (None, "A")),
+    ("ljust", (None, 12)),
+    ("lstrip", (None, "A")),
     ("replace", (None, "A", "B")),
     ("rfind", (None, "A")),
     ("rindex", (None, "e")),
-    pytest.param("rjust", (None, 12), marks=unicode_bug_fail),
-    ("rpartition", (None, "A")),
+    ("rjust", (None, 12)),
```

Comment:
If you remove `ljust` and `rjust` from this listing I think all the tests will pass. We didn't backport the text padding ufuncs.